ReactWindows/ReactNative.Net46/Tracing/NullLoggingActivityBuilder.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+
+namespace ReactNative.Tracing
+{
+    /// <summary>
+    /// Temporary NullLoggingActivityBuilder.
+    /// </summary>
+    public class NullLoggingActivityBuilder
+    {
+        /// <summary>
+        /// Dummy method to satisfy interface requirements.
+        /// </summary>
+        /// <returns>An empty disposable object.</returns>
+        public IDisposable Start()
+        {
+            return new NullDisposable();
+        }
+
+        /// <summary>
+        /// Empty disposable object.
+        /// </summary>
+        class NullDisposable : IDisposable
```

Comment:
Doesn't matter, up to you.  Less to maintain in the long term if you remove it.

---
In reply to: [82057270](https://github.com/ReactWindows/react-native-windows/pull/740#discussion_r82057270) [](ancestors = 82057270)